packages/modules/aave-v3/src/actions/deployAggregatorBandAdapter.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from 'zod'
+
+import { Action, InfinitWallet, SubAction } from '@infinit-xyz/core'
+import { validateActionData, zodAddressNonZero } from '@infinit-xyz/core/internal'
+
+import {
+  DeployAggregatorBandAdapterSubAction,
+  DeployAggregatorBandAdapterSubActionParams,
+} from '@actions/subactions/deployAggregatorBandAdapter'
+
+export const DeployAggregatorBandAdapterActionParamsSchema = z.object({
+  ref: zodAddressNonZero,
+  aggregatorBandAdapterConfigs: z
+    .array(
+      z.object({
+        name: z.string().describe('Name of the price pair'),
+        base: z.string().describe('String of the base'),
```

Comment:
we need the exact base and quote for using Band Oracle so I think we should add Band Docs docs and examples here to clarify the user